qutebrowser/config/configdata.yml

```diff
@@ -3,8 +3,9 @@
 aliases:
   default:
     w: session-save
-    q: quit
-    wq: quit --save
```

Comment:
`wq` is gone now which may break peoples workflows completely. Is that what you intended? Should it be something like `close ;; save`?

Reply:
IIRC `:save` doesn't save the session, so that would be different. If anything, `:wq` should still be `:quit --save` probably, but then `:q` does something different to `:wq` which sucks as well - so I proposed in IRC to just get rid of it.

Reply:
Would it be possible to have `:wq` save the session and close the window *if there is only one window* and issue a warning/message *if there are several windows* pointing to `:wqa`? After all, as long as there is only a single window, `:q` is identical to `:qa`.

Reply:
We could add a `--save` argument to `:close`, maybe?